mlflow/pyfunc/__init__.py

```diff
@@ -1096,6 +1152,38 @@ def _predict_row_batch(predict_fn, args):
             StringType: np.str_,
         }
 
+        def _convert_array_values(values, spark_type):
+            array_dim = 1
+            elem_type = spark_type.elementType
+            if isinstance(elem_type, ArrayType):
+                elem_type = elem_type.elementType
+                if isinstance(elem_type, ArrayType):
+                    raise MlflowException(
+                        f"Unsupported array<array<array<ElementType>>> type field "
+                        "in struct type."
```

Comment:
I think `array<array<array<ElementType>>>` is confusing. Can we update the message?

Reply:
What's the better description ?

Reply:
I think you should mention the following in the error message.

1. What's supported.
2. Actual type.